### DISCUSSION TITLE
Secondary like/dislike count separate from normal that is only from users of this extention

### DISCUSSION THREAD

**Discussion by ghost** [Anarios/return-youtube-dislike#494] [Ideas]
It would be cool to have a private like/dislike count just for people who use the extension. This could be displayed under or next to the main dislike count. I know most of the time there would be no likes/dislike from people because it is not super popular yet but I would still love to see it.

**Comment by hazzarr1** [comment]
float the website to creators who want the dislike back, get followers to install until youtube brings it back

**Comment by cyrildtm** [comment]
see #530

**Comment by k98kurz** [comment]
If this is to be a serious idea, then some implementation ideas are necessary. I know there have been discussions previously about this exact topic before Github discussions were created because Google announced that their API showing the like/dislike ratio will be deprecated and removed. [I propose using CRDTs to accomplish this for good scalability, availability, and strong eventual consistency](https://github.com/Anarios/return-youtube-dislike/discussions/707).
